ServerSide/yoty/WebApi/PublicDataSchemas/DTO/BidDTO.cs

```diff
@@ -1,36 +1,43 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿// Copyright (c) YOTY Corporation and contributors. All rights reserved.
 
 namespace YOTY.Service.WebApi.PublicDataSchemas
 {
+    using System;
+
     // data structure represents product bit with crud
     public class BidDTO: BaseDTO
     {
-        string Id { get; set; }
-
-        string Name { get; set; }
+        public string Id { get; set; }
 
-        string OwnerId { get; set; }
+        public ProductDTO ProductDto { get; set; }
```

Comment:
I think this means our db should also separate between product and bid, right? I can add it in my next PR

Reply:
@yotamc4  yes we probably going to have many to one relation between bids and product